djimporter/static/djimporter/js/upload_status.js

```diff
@@ -0,0 +1,24 @@
+let current_status = $('#js-data').data('importlog-status');
+if (['running', 'created'].includes(current_status)){
+    getData();
+}
+
+function getData() {
+    let pk = $('#js-data').data('importlog-pk');
+    //$('#js-data').data('import-log-get')
+    let url = window.location + 'get/';
```

Comment:
This solution allows us to overwrite get method sometimes. I can't put and ovewrite url_get method using content in DetailView

Reply:
```suggestion
    // TODO: add importlog-url attr
    let url = $('#js-data').data('importlog-url');
```